conductr_cli/bndl_main.py

```diff
@@ -364,6 +369,19 @@ def add_conf_arguments(parser):
                              'Example: bndl --volume my-vol=/data --component web',
                         metavar='NAME=MOUNT-POINT')
 
+    parser.add_argument('--with-defaults',
+                        default=None,
```

Comment:
By default, `with_defaults` is empty. Only for the format `oci_image` and `docker` we automatically set `with_defaults` to `generic` if it has not been specified with `--with-defaults`. Are you suggesting that we should move the logic of setting it to `generic` for oci and docker formats to the `bndl_main` module, e.g. to https://github.com/markusjura/conductr-cli/blob/bndl-defaults/conductr_cli/bndl_main.py#L156-L157